app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def _variable_sorter(*args) -> dict:
+
+    result = {"mutable": [], "immutable": [], }
+
+    for var_name in args:
+        if (isinstance(var_name, set) or isinstance(var_name, dict)
+                or isinstance(var_name, list)):
+            result["mutable"].append(var_name)
+            continue
+        result["immutable"].append(var_name)
+
+    return result
+
+
+sorted_variables = _variable_sorter(
+    lucky_number, pi, one_is_a_prime_number, marks,
+    name, my_favourite_films, profile_info, collection_of_coins
+)
```

Comment:
```suggestion

sorted_variables = {
    "immutable": [lucky_number ... ],
    "mutable": [marks ...]
}
```

Reply:
it's good that you practiced, and your solution is good, 
but here it's unnecessary, you can do it just like I showed

Reply:
![image](https://github.com/mate-academy/py-mutable-immutable/assets/93856423/4b493330-fd31-4cd0-8d67-01a5e8cafb58)

Reply:
I know that simple is better than comlex, but handle assignment of variables is similar to the Python Basic level, not Python core. Let it be)